tests/integration/devfile/cmd_devfile_catalog_test.go

```diff
@@ -92,4 +92,18 @@ var _ = Describe("odo devfile catalog command tests", func() {
 			helper.MatchAllInOutput(output, wantOutput)
 		})
 	})
+
+	Context("When executing catalog list components with registry that is not set up properly", func() {
+		It("should list components from valid registry", func() {
+			helper.CmdShouldPass("odo", "registry", "add", "fake", "http://fake")
+			output := helper.CmdShouldPass("odo", "catalog", "list", "components")
+			wantOutput := []string{
```

Comment:
How about doing this in a single line like https://github.com/openshift/odo/blob/master/tests/integration/devfile/cmd_devfile_push_test.go#L197 ?

Reply:
Done